src/lib/utils.ts

```diff
@@ -1,3 +1,10 @@
+import {
+  NFTHISTORY_API_URL,
+  NFTLIST_API_URL,
+  NFTTRANSFER_API_URL,
```

Comment:
Maybe its time to use `API_URL` only and use it as base path, like:

```
- ${API_URL}/history
- ${API_URL}/nfts
- ${API_URL}/transfers
- etc.
```

instead of adding each endpoint as an export